app/views/datasets/_additional_info.html.erb

```diff
@@ -72,9 +72,9 @@
             <% if i['harvest_object_id'] %>
               <dt><%= t('.inspire_gemini_record') %></dt>
               <dd>
-                <%= link_to t('.xml'), "/api/2/rest/harvestobject/#{ i['harvest_object_id'] }/xml", class: 'govuk_link' %>
+                <%= link_to t('.xml'), "/api/2/rest/harvestobject/#{ i['harvest_object_id'] }/xml", class: 'govuk-link' %>
```

Comment:
Non-blocker: I'm aware this wasn't introduced by this change, but I would suggest we improve the semantics too by using two `dd`s instead of one split using a `br`
```erb
<dt><%= t('.inspire_gemini_record') %></dt>
<dd><%= link_to t('.xml'), "/api/2/rest/harvestobject/#{ i['harvest_object_id'] }/xml", class: 'govuk-link' %></dd>
<dd><%= link_to t('.html'), "/api/2/rest/harvestobject/#{ i['harvest_object_id'] }/html", class: 'govuk-link' %></dd>
```
It may require some additional styling.